src/modules/ejector/ejector.py

```diff
@@ -261,8 +275,7 @@ def _get_predicted_withdrawable_epoch(
 
         churn_limit = self._get_churn_limit(blockstamp)
 
-        remain_exits_capacity_for_epoch = churn_limit - latest_to_exit_validators_count
-        epochs_required_to_exit_validators = (validators_to_eject_count - remain_exits_capacity_for_epoch) // churn_limit + 1
+        epochs_required_to_exit_validators = (validators_to_eject_count + latest_to_exit_validators_count) // churn_limit
```

Comment:
still not sure whether it really needs to change

Reply:
The result is the same, I just simplified the expression

Reply:
you can do it on paper and check it

Reply:
I'm here against any changes in logic, even simplification of expressions